### ISSUE TITLE
[Feature Request] Process page through add-ons like regular window does

### ISSUE THREAD

**Issue by git6578** [mozilla/side-view#286]
Great add-on idea! Will definetly try again soon.

ISSUE:
Pages get loaded raw and unfiltered.
One can not block any content like javascripts, XHR, ..., in the sidebar.
Tested with uMatrix add-on and reddit.com

Please make pages be processed like regular window pages, through add-ons etc.

(additionally: Fullscreen functions do neither work for the side bar nor the full screen)

**Comment by johngruen** [comment]
Yeah this would be cool, related to underlying APIs and #288 and #289

**Comment by Tiepi** [comment]
I heavily rely on the Firefox multi-account containers add-on. If pages aren't processed through this add-on, the result is that I have to login again on almost every site I send to the Side View.

**Comment by ExE-Boss** [comment]
This might end up being fixed as a result of using `<iframe mozbrowser>` (see #327).
